share/locale/en.json

```diff
@@ -1,165 +1,165 @@
-    "update_error_check_error": "Error checking for updates: Maybe you're not connected to Tor, or maybe the OnionShare website is down.",
-    "update_error_invalid_latest_version": "Error checking for updates: The OnionShare website responded saying the latest version is '{}', but that doesn't appear to be a valid version string.",
-    "update_not_available": "You are running the latest version of OnionShare.",
-    "gui_tor_connection_ask": "Would you like to open OnionShare settings to troubleshoot connecting to Tor?",
-    "gui_tor_connection_ask_open_settings": "Open Settings",
+    "update_available": "New version of OnionShare. <a href='{}'>Click here</a> to get it.<br><br>You have {} and the latest is {}.",
+    "update_error_check_error": "Could not check for new versions: Maybe you're not connected to Tor, or the OnionShare website is down?",
+    "update_error_invalid_latest_version": "Could not check for new versions: The OnionShare website is saying the latest version is the unrecognizable '{}'…",
+    "update_not_available": "You are running the latest OnionShare.",
+    "gui_tor_connection_ask": "Would you like to open the settings to troubleshoot connecting to Tor?",
+    "gui_tor_connection_ask_open_settings": "Yes",
     "gui_tor_connection_ask_quit": "Quit",
-    "gui_tor_connection_error_settings": "Try adjusting how OnionShare connects to the Tor network in Settings.",
-    "gui_tor_connection_canceled": "OnionShare could not connect to Tor.\n\nMake sure you're connected to the Internet, then re-open OnionShare to set up the Tor connection.",
+    "gui_tor_connection_error_settings": "Try changing how OnionShare connects to the Tor network in the settings.",
+    "gui_tor_connection_canceled": "Could not connect to Tor.\n\nEnsure you are connected to the Internet, then re-open OnionShare and set up its connection to Tor.",
     "gui_tor_connection_lost": "Disconnected from Tor.",
-    "gui_server_started_after_timeout": "The server started after your chosen auto-timeout.\nPlease start a new share.",
-    "gui_server_timeout_expired": "The chosen timeout has already expired.\nPlease update the timeout and then you may start sharing.",
-    "share_via_onionshare": "Share via OnionShare",
+    "gui_server_started_after_timeout": "The expiry timer ran out before the server started.\nPlease start a new share.",
+    "gui_server_timeout_expired": "The expiry timer ran out.\nPlease restart it to start sharing.",
+    "share_via_onionshare": "OnionShare it",
     "gui_save_private_key_checkbox": "Use a persistent address\n(unchecking will delete any saved addresses)",
-    "gui_url_description": "<b>Anyone</b> with this link can <b>download</b> your files using the <b>Tor Browser</b>: <img src='{}' />",
-    "gui_url_label_persistent": "This share will not expire automatically unless a timer is set.<br><br>Every share will have the same address (to use one-time addresses, disable persistence in Settings)",
```

Comment:
Unless I am mistaken, it only says "Use a persistent address"…?

Reply:
"This share will not expire automatically unless a timer is set.<br><br>Every subsequent share will be available at the same address (to use one-time addresses, turn off \"Use a persistent address\" in the settings)"